django_fastdev/apps.py

```diff
@@ -243,11 +251,16 @@ def is_from_project(cls):
     if not module or not hasattr(module, "__file__"):
         return False
 
-    venv_dir = os.environ.get("VIRTUAL_ENV", "")
+    venv_dir = get_venv_path()
+    project_dir = get_path_for_django_project()
     module_path = os.path.abspath(module.__file__)
-    return module_path.startswith(
-        str(settings.BASE_DIR)
-    ) and not module_path.startswith(venv_dir)
+
+    if module_path.startswith(str(project_dir)):
+        # check against venv after project dir matches,
+        # just in case venv resides in project dir
+        if venv_dir and module_path.startswith(str(venv_dir)):
+            return False
+        return True
```

Comment:
This new code is not what I suggested, and it's broken. It implicitly returns `None`.

Reply:
I'm confused, you me the `if venv_dir` bit? I just use implicit truthy testing in case somehow venv_dir is an empty string (using `is None` would cause that check to pass, and then `startswith()` would always succeed since it is being passed a blank string)

Reply:
No, I mean your code is like:
```py
def foo():
    if x: 
        return True
   # NO ELSE HERE
```
That's the problem.

Reply:
ohhh, duh! fixed, sorry! :P